src/calip/core.clj

```diff
@@ -151,13 +155,14 @@
   ([fs]
    (trace fs {}))
   ([fs opts]
-     (doseq [f (unwrap-stars fs)]
-       (let [fvar (f-to-var f)]
-         (println "wrapping" fvar "in µ/trace")
-         (hooke/add-hook fvar                                ;; target var
-                         (str fvar)                          ;; hooke key
-                         (partial make-trace opts fvar))))
-     (unwrap-stars fs)))
+   (doseq [f (unwrap-stars fs)]
+     (let [fvar (f-to-var f)]
+       (when-not *silent*
+         (println "wrapping" fvar "in µ/trace"))
+       (hooke/add-hook fvar                                ;; target var
+                       (str fvar)                          ;; hooke key
+                       (partial make-trace opts fvar))))
+   (unwrap-stars fs)))
```

Comment:
leave the original formatting

Reply:
This one I can't get to save in the original formatting.  It doesn't match cljfmt or intellij.  I split the formatting to its own commit.